schemas/context/webpage.schema.json

```diff
@@ -50,17 +50,12 @@
           "type": "object",
           "description": "The referrer of the web page.",
           "properties": {
-            "URL": {
+            "xdm:URL": {
               "title": "URL",
               "type": "string",
               "description": "The referrer URL."
             },
-            "domain": {
-              "title": "Domain",
-              "type": "string",
-              "description": "The domain of the referrer."
-            },
-            "type": {
+            "@type": {
               "title": "Type",
               "type": "string",
               "description": "The referrer type."
```

Comment:
What are the possible values here?  Do they come from an existing standard?  If so, please cite.